### ISSUE TITLE
New room - When adding room name & use Space "-" is displayed between words instead of "_"

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#22314]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Navigate to staging.new.expensify.com
2. Click on the green + button
3. Click on the "New Room" option
4. Enter room name that contain few words & use "Space" to separate the words (e.g #hello room name)

## Expected Result:
When entering the room name, entering a "Space" should add a "_" to the name (e.g #hello_room_name)

## Actual Result:
When entering the room name, entering a "Space" adds a "-" to the name (e.g #hello-room-name)

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.37.2

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/78819774/1c751605-2e83-4885-b261-c028af87d189

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by trjExpensify** [comment]
Yeah, we use hyphens instead of underscore for room names. CC: @MitchExpensify 

We need to update this [test case](https://expensify.testrail.io/index.php?/cases/view/1971088&group_by=cases:section_id&group_order=asc&display_deleted_cases=0&group_id=229067), specifically this line: 

>Verify that when entering the room name, entering a "Space" adds a "_" to the name (e.g #hello_room_name)

To 

>Verify that when entering the room name, entering a "Space" adds a "-" to the name (e.g #hello-room-name)

**Comment by trjExpensify** [comment]
It's a minor edit, I've done that directly. Closing this out! CC: @isagoico for vis.
